### PR TITLE
feat: adds small-check-icon

### PR DESCRIPTION
**Pull request by sooinc**
### PR Checklist

- [x] Related to designs: specific design for weekly target needed a small version of the check mark and scaling the current checkmark did not work visually so adding a customized smallCheckIcon.
- [x] Related to JIRA ticket: https://codecademy.atlassian.net/browse/EGG-256?atlOrigin=eyJpIjoiOGU3YTYwNDBmZDI2NDc1NmFlODY4NjFjMDdhNjA5NDkiLCJwIjoiaiJ9 
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.8.3-alpha.5cae52.0<br/>@codecademy/gamut-labs@4.2.11-alpha.5cae52.0<br/>@codecademy/gamut@16.1.6-alpha.5cae52.0<br/>@codecademy/styleguide@17.6.6-alpha.5cae52.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f871cf368d5b11ebc8c05f1--gamut-preview.netlify.app>https://5f871cf368d5b11ebc8c05f1--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f871cf368d5b11ebc8c05f1>Deploy Logs</a></p>

File: packages/gamut-icons/src/svg/small-check-icon.svg [hunk 1]
```diff
@@ -0,0 +1,12 @@
+    <svg
+      width="20"
+      height="20"
+      viewBox="0 0 8 7"
```
[the remark this answers is not in the document]
Comment (sooinc) [reply packages/gamut-icons/src/svg/small-check-icon.svg:4]: Yes! I updated my PR overview with additional context on where it's being used and why we needed another one.

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-icons@3.8.3-alpha.946722.0<br/>@codecademy/gamut-labs@4.3.1-alpha.946722.0<br/>@codecademy/gamut@16.2.1-alpha.946722.0<br/>@codecademy/styleguide@17.7.1-alpha.946722.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5f874ccb77884757435de3e2--gamut-preview.netlify.app>https://5f874ccb77884757435de3e2--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/5f874ccb77884757435de3e2>Deploy Logs</a></p>

**Pull request merged**